### PR TITLE
Remove duplicate next link in README

### PR DESCRIPTION
**Pull request by Zeedinstein**
Noticed a duplicate link in the readme that was unnecessary.

### CODE REVIEW

**Review by jpage-godaddy — APPROVED** [review]

**Comment by jpage-godaddy** [comment]
Is this auto-generated code, though? I wonder how it got generated as a duplicate.

**Review by jpage-godaddy — CHANGES_REQUESTED** [review]
Actually, this is for sure generated code. Looks like the reason it was duplicated is here:

https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-nextjs/lib/index.js#L242-L265

**Comment by Zeedinstein** [comment]
Sure I'll have a look at that

**Comment by Zeedinstein** [comment]
@jpage-godaddy This should now only generate the one next link. Which I don't see a problem with. Unless having 2 links was done on purpose for express and fastify? But I still don't see a reason to have 2 next links.

**Comment by jpage-godaddy** [comment]
> @jpage-godaddy This should now only generate the one next link. Which I don't see a problem with. Unless having 2 links was done on purpose for express and fastify? But I still don't see a reason to have 2 next links.

Hm, yeah I didn't notice until now that they had different parents, so they're not really duplicated. When the lifecycle diagram is generated, those two `next` entries are both used. It sounds like the script that generates the markdown documentation is the real issue; that needs to do the de-duplication. Finding where that bug in the documentation generation code is would be key here.

File: packages/gasket-plugin-nextjs/lib/index.js [hunk 1]
```diff
@@ -255,14 +255,6 @@ module.exports = {
           parent: 'express',
           after: 'next'
         },
-        {
-          name: 'next',
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-nextjs/lib/index.js:259]: Good catches and thanks for being willing to contribute @Zeedinstein. As @jpage-godaddy mentioned, we would still need this entry for the graph generation.

So yea, a couple issues... we should probably only generated one or the other depending on which, either Express or Fastify, Gasket plugins are in use by an app when using the `gasket docs` command. However, that still will not address the duplication in the generated index readme for the repo here. I think the duplication will be necessary, with the best thing to do being to clarify things in the descriptions:
- `Update the Next.js app instance before preparing for Express`
- `Update the Next.js app instance before preparing for Fastify`

**Comment by agerard-godaddy** [comment]
Resolved in https://github.com/godaddy/gasket/pull/472

**Pull request closed without merging**